util/dvsim/testplanner/README.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: "Testplanner tool"
+title: "dv_planner tool"
```

Comment:
I think better to revert all changes in this area to avoid tool breakage.

Reply:
Actually on second thoughts, these changes are in doc, so there wont be tool breakages I suppose.

Reply:
Reverted all these.. I thought I already did. but apparently I missed the README